### DISCUSSION TITLE
Windows binaries on arm64

### DISCUSSION THREAD

**Discussion by woachk** [ciderapp/Cider#1424] [Feature Request]
Hello,

On Windows this seems to currently run through the compatibility layer.

https://www.electronjs.org/docs/latest/tutorial/windows-arm/

**Comment by cryptofyre** [comment]
Unfortunately due to castlabs not creating builds for arm64 Windows platforms which we depend on due to Widevine constraints this is not currently within our active scope.

I'd highly suggest making an issue on their GitHub page here:
https://github.com/castlabs/electron-releases/issues

**Reply by tva2000hn** [reply]
Hmm , Cider 1 port to WV2?
